.env.example

```diff
@@ -1,13 +1,12 @@
-NEXT_PUBLIC_CRE8ORS_ADDRESS=
-NEXT_PUBLIC_ALLOW_LIST_ADDRESS=
-NEXT_PUBLIC_CHAIN_ID=
-NEXT_PUBLIC_ALLOW_LIST_CHAIN_ID=
-NEXT_PUBLIC_OPENZEPPELIN_URL=
-NEXT_PUBLIC_ALLOWLIST_API_KEY=
-MONGODB_URI=
-DEFENDER_AUTOTASK_WEBHOOK=
 TWITTER_CONSUMER_KEY=
 TWITTER_CONSUMER_SECRET=
 TWITTER_ACCESS_TOKEN_KEY=
 TWITTER_ACCESS_TOKEN_SECRET=
+TWITTER_BEARER_TOKEN=
+OPENZEPPLIN_FORWARDER_ADDRESS=
+DEFENDER_AUTOTASK_WEBHOOK=
 OPENAI_API_KEY=
+ALCHEMY_RPC_URL=
+NEXT_PUBLIC_CONTRACT_ADDRESS=
+NEXT_PUBLIC_NETWORK=
+ETHERSCAN_NETWORK=
```

Comment:
need to add `NEXT_PUBLIC_NETWORK` and `ETHERSCAN_NETWORK` to env variables before testing.
for preview and dev
```NEXT_PUBLIC_NETWORK=mumbai
ETHERSCAN_NETWORK=maticmum```
for prod
```NEXT_PUBLIC_NETWORK=polygon
ETHERSCAN_NETWORK=matic```